README.md

```diff
@@ -580,6 +581,20 @@ Address.record_limit(10_000).batch(100).each { … } # Batch specified as part o
 The implication of batches is that the underlying requests are done in the batch sizes to make the request and responses
 more manageable. Note that this batching is for `Query` and `Scans` and not `BatchGetItem` commands.
 
+##### DynamoDB Native Pages
+At times it can be useful to rely on DynamoDB [default pages](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.html#Query.Pagination)
+instead of fixed pages sizes. Each page results in a single Query or Scan call
+to DyanmoDB, but returns an unknown number of records.
+
+Access to the native DynamoDB pages can be obtained via the `find_by_pages`
+method, which yields arrays of records.
+
+```ruby
+Address.find_by_pages do |addresses|
+  # have an array of pages
+end
+```
```

Comment:
My next PR adding the restart hashes will continue with this section for both fixed limit and native pages.